.github/workflows/rust.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
-      - run: rustup component add rustfmt
```

Comment:
more straightforward to just use the action's configuration, instead of doing things as an extra manual step